app/api/routes/root.js

```diff
@@ -0,0 +1,11 @@
+"use strict";
+
+var Gist = require("../gist/index");
+
+module.exports = function route(request, response) {
+	var gist = new Gist(request.params.gistId);
+	gist.blobs()
+	.then(function (blobs) {
```

Comment:
I still prefer an indent here. Makes it more obvious that the then belongs to the above line.